java-checks/src/main/java/org/sonar/java/checks/SuppressWarningsCheck.java

```diff
@@ -51,11 +50,19 @@
 
   @RuleProperty(
     key = "listOfWarnings",
-    description = "Comma separated list of warnings that can't be suppressed. Example: 'unchecked, cast, all, boxing'. An empty list means that no warning can be suppressed.",
+    description = "Comma separated list of warnings that can be suppressed (example: unchecked, cast, boxing). An empty list means that no warning can be suppressed.",
     defaultValue = "")
-  public String warningsCommaSeparated = "";
+  private String warningsCommaSeparated = "";
```

Comment:
In order to use the same approach as for all the other checks using properties and to guarantee any future development, you should let the property as `public` member (and consequently remove constructors and adapt tests).